### PR TITLE
[docs] add ipustrategy Hyperlink

### PR DESCRIPTION
**Pull request by gouzil**
### PR types

Bug fixes

### PR changes

Docs

### Describe

add ipustrategy Hyperlink

cn docs: https://github.com/PaddlePaddle/docs/pull/5261

### CODE REVIEW

**Comment by Ligoml** [comment]
image

可以参考这个commit的修改，能正确显示API的索引：https://github.com/PaddlePaddle/Paddle/pull/46422/commits/68a4273a6568a4eab40458b0d4aa4eede2cbf672

**Review by Ligoml — DISMISSED** [review]
LGTM

**Review by Ligoml — DISMISSED** [review]
LGTM @SigureMo 你手是真的快

**Comment by SigureMo** [comment]
嘿嘿(๑>؂<๑）

File: python/paddle/tensor/manipulation.py [hunk 1]
```diff
@@ -4333,8 +4333,9 @@ def put_along_axis(arr, indices, values, axis, reduce='assign'):
         indices (Tensor) : Indices to put along each 1d slice of arr. This must match the dimension of arr,
             and need to broadcast against arr. Supported data type are int and int64.
         axis (int) : The axis to put 1d slices along.
-        reduce (string | optinal) : The reduce operation, default is 'assign', support 'add', 'assign', 'mul' and 'multiply'.
-    Returns :
+        reduce (string, optinal) : The reduce operation, default is 'assign', support 'add', 'assign', 'mul' and 'multiply'.
```
Comment (SigureMo) [inline python/paddle/tensor/manipulation.py]:
```suggestion
        reduce (str, optional): The reduce operation, default is 'assign', support 'add', 'assign', 'mul' and 'multiply'.
```

[doge]

**Review by Ligoml — APPROVED** [review]

**Review by zhiqiu — APPROVED** [review]
LGTM

**Pull request merged**